harvest/broker/_base.py

```diff
@@ -11,8 +12,8 @@
 import pytz
 
 # Submodule imports
-import harvest.load
-
+from harvest.utils import is_crypto
+from harvest.storage import BaseStorage
```

Comment:
load will be removed shortly by storage. Also added a utils file for random functions used throughout the codebase.